mautrix_facebook/example-config.yaml

```diff
@@ -91,6 +91,7 @@ bridge:
     # "own_nickname" (user-specific!)
     displayname_preference:
     - name
+    - first_name
```

Comment:
Are there cases where `first_name` is set, but `name` isn't?

Reply:
Maybe I did something wrong, but I had most of my contacts being `fb_<their id> (FB)` and a few with `Full name (FB)`.